OpenSim/Common/Sine.h

```diff
@@ -135,11 +104,18 @@ OpenSim_DECLARE_CONCRETE_OBJECT(Sine, Function);
     int getArgumentSize() const override {return 1;}
     int getMaxDerivativeOrder() const override {return 10;}
 
+private:
+    void constructProperties() {
+        constructProperty_amplitude(1.0);
+        constructProperty_omega(2 * SimTK::Pi);
```

Comment:
I think we should leave the default value as it was. I actually think the name for this property is not correct; usually `f = 2*pi*omega`, so this property should be called "frequency" (as there is no `2*pi` in `calcValue()`). Changing `omega`'s default to `2*pi` is even more confusing, I think. It also makes sense that default properties just give `sin(x)`. Consider that this function is not necessarily only used for functions of time; sin(2*pi*(joint_angle_in_radians))` doesn't make much sense.

This change is also backwards-incompatible (I wouldn't say that the mechanism we have for preserving defaults in a backwards compatible way, via the `defaults` node in XML files, is fully functional).